sretoolbox/container/image.py

```diff
@@ -380,7 +380,8 @@ def _request_get(self, url, method=requests.get):
         else:
             auth = self.auth
 
-        response = method(url, headers=headers, auth=auth)
+        response = method(url, headers=headers, auth=auth,
+                          verify=not self.registry.startswith('127.0.0.1'))
```

Comment:
Localhost is not supposed to be that special. We should add a new param to the `Image` class, `ssl_verify=True` and set it to false (`i = Image(..., ssl_verify=False`) when needed.

Reply:
@apahim done